lib/BaseSlackHandler.js

```diff
@@ -48,15 +48,15 @@ BaseSlackHandler.prototype.getSlackRoomNameFromID = async function(id, token) {
     return id;
 };
 
-BaseSlackHandler.prototype.replaceChannelIdsWithNames = async function(message, token) {
+BaseSlackHandler.prototype.replaceChannelIdsWithNames = async function(message, text, token) {
     var main = this._main;
 
-    if (message.text === undefined) {
+    if (text === undefined) {
         return message;
     }
 
     // match all channelIds
-    var testForName = message.text.match(CHANNEL_ID_REGEX);
+    var testForName = text.match(CHANNEL_ID_REGEX);
```

Comment:
[var](https://media.tenor.com/images/50c5ec11ca69d065be8715b9950f41eb/tenor.gif)

Reply:
Do you just want a bigger diff to update to ts? :p

Reply:
*grumbles* okay fine, I already fixed this in TS